sources/engine/Stride.UI.Tests/Layering/EditTextTests.cs

```diff
@@ -17,7 +17,7 @@ namespace Stride.UI.Tests.Layering
     [System.ComponentModel.Description("Tests for EditText layering")]
     public class EditTextTests
     {
-        class DummyFont : SpriteFont { }
+        public class DummyFont : SpriteFont { }
```

Comment:
`AssemblyScanProcessor: Can't register type [Stride.UI.Tests.Layering.EditTextTests/DummyFont] for scan type [Stride.Core.Serialization.Contents.ReferenceSerializerAttribute] because it is a nested private type`